src/pg/relations.cpp

```diff
@@ -177,4 +177,19 @@ GetOid(Form_pg_class rel) {
 	return rel->oid;
 }
 
+namespace pg {
+
+Form_pg_attribute
+GetAttributeByName(TupleDesc tupdesc, const char *colname) {
```

Comment:
`GetAttributeByName` also existed as a function in PG14/15 (and that function has different behaviour). We now put it in a namespace to avoid this naming collision.